doc/en/cache.rst

```diff
@@ -266,3 +266,13 @@ dumps/loads API of the json stdlib module
 .. automethod:: Cache.get
 .. automethod:: Cache.set
 .. automethod:: Cache.makedir
+
+
+New tests first
+-----------------
+
+The plugin provides command line options to run tests in another order:
```

Comment:
I think this should be moved closer to `--lf`, right before the `The new config.cache object` section.

Reply:
I also think we should use the same blurb from the changelog. 👍